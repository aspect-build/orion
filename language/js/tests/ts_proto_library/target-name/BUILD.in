# gazelle:js_proto_naming_convention ts__{proto_library}
