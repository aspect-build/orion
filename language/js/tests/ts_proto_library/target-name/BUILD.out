load("@aspect_rules_ts//ts:proto.bzl", "ts_proto_library")
load("@rules_proto//proto:defs.bzl", "proto_library")

# gazelle:js_proto_naming_convention ts__{proto_library}

proto_library(
    name = "foo_proto",
    srcs = ["foo.proto"],
    visibility = ["//visibility:public"],
)

ts_proto_library(
    name = "ts__foo_proto",
    node_modules = "//:node_modules",
    proto = ":foo_proto",
)
